test/e2e/e2eutil.go

```diff
@@ -24,24 +23,29 @@ import (
 	k8sClient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	DeployDir   = "/go/deploy"
```

Comment:
it looks like this is only being used in the `setup` package, can we unexport it and move it there?

Reply:
Done!